crates/stc_ts_type_checker/tests/conformance.pass.txt

```diff
@@ -2326,6 +2327,8 @@ types/typeParameters/typeParameterLists/typeParametersAvailableInNestedScope.ts
 types/typeParameters/typeParameterLists/typeParametersAvailableInNestedScope2.ts
 types/typeParameters/typeParameterLists/typeParametersAvailableInNestedScope3.ts
 types/typeParameters/typeParameterLists/typesWithDuplicateTypeParameters.ts
+types/typeRelati
```

Comment:
Please don't commit after terminating `check.sh`

Reply:
I'm sorry.
It's unintentional.